dockstore-common/src/main/java/io/dockstore/common/yaml/DockstoreYamlHelper.java

```diff
@@ -148,9 +153,13 @@ public Property getProperty(Class<?> type, String name) {
     }
 
     static DockstoreYaml readDockstoreYaml(final String content) throws DockstoreYamlException {
```

Comment:
Method looks to be not invoked any more, I think you can delete it.

Reply:
Deleted.